src/brr.c

```diff
@@ -4,31 +4,32 @@
 #include <stdlib.h>
 #include "ebmusv2.h"
 
-enum {
-	BRR_BLOCK_SIZE = 9,
+const unsigned int BRR_BLOCK_SIZE = 9;
```

Comment:
This was originally changed so it could be shared between files. I've changed it back so it's in an enum in both files and we can sort out using a shared enum later hopefully during a project-wide refactor.